kafka-connect-s3/src/test/java/io/confluent/connect/s3/DataWriterAvroTest.java

```diff
@@ -75,6 +79,9 @@
 import static org.junit.Assert.assertThat;
 import static org.junit.Assert.assertTrue;
 
+@RunWith(PowerMockRunner.class)
```

Comment:
Required because this uses the class S3SinkConnectorTestBase which mocks a private static method.